src-plugins/itkDataImage/readers/itkDCMTKDataImageReader.cpp

```diff
@@ -357,117 +357,104 @@ bool itkDCMTKDataImageReader::readInformation (const QStringList& paths)
-    performer      << d->io->GetPerformingPhysicianName().c_str();
-    institution    << d->io->GetInstitution().c_str();
-    protocol       << d->io->GetProtocolName().c_str();
-    comments       << d->io->GetAcquisitionComments().c_str();
-    status         << d->io->GetPatientStatus().c_str();
-    report << "";
-
-    for (unsigned int i=0; i<d->io->GetOrderedFileNames().size(); i++ )
-      filePaths << d->io->GetOrderedFileNames()[i].c_str();
-
-    if (!medData->hasMetaData(medMetaDataKeys::PatientName.key() ))
-      medData->addMetaData(medMetaDataKeys::PatientName.key(), patientName );
-    else
-      medData->setMetaData(medMetaDataKeys::PatientName.key(), patientName );
-
-    if (!medData->hasMetaData(medMetaDataKeys::StudyDescription.key() ))
-      medData->addMetaData(medMetaDataKeys::StudyDescription.key(), studyName );
-    else
-      medData->setMetaData(medMetaDataKeys::StudyDescription.key(), studyName );
-
-    if (!medData->hasMetaData (medMetaDataKeys::SeriesDescription.key() ))
-      medData->addMetaData(medMetaDataKeys::SeriesDescription.key(), seriesName );
-    else
-      medData->setMetaData(medMetaDataKeys::SeriesDescription.key(), seriesName );
-
-    medData->setMetaData(medMetaDataKeys::StudyDicomID.key(),    studyDicomId);
-    medData->setMetaData(medMetaDataKeys::SeriesDicomID.key(),   seriesDicomId);
-    medData->setMetaData(medMetaDataKeys::Orientation.key(),     orientation);
-    medData->setMetaData(medMetaDataKeys::SeriesNumber.key(),    seriesNumber);
-    medData->setMetaData(medMetaDataKeys::SequenceName.key(),    sequenceName);
-    medData->setMetaData(medMetaDataKeys::SliceThickness.key(),  sliceThickness);
-    medData->setMetaData(medMetaDataKeys::Rows.key(),            rows);
-    medData->setMetaData(medMetaDataKeys::Columns.key(),         columns);
-    medData->setMetaData(medMetaDataKeys::Age.key(),             age);
-    medData->setMetaData(medMetaDataKeys::BirthDate.key(),       birthdate);
-    medData->setMetaData(medMetaDataKeys::Gender.key(),          gender);
-    medData->setMetaData(medMetaDataKeys::Description.key(),     desc);
-    medData->setMetaData(medMetaDataKeys::Modality.key(),        modality);
-    medData->setMetaData(medMetaDataKeys::AcquisitionDate.key(), acqdate);
-    medData->setMetaData(medMetaDataKeys::Referee.key(),         referee);
-    medData->setMetaData(medMetaDataKeys::Performer.key(),       performer);
-    medData->setMetaData(medMetaDataKeys::Institution.key(),     institution);
-    medData->setMetaData(medMetaDataKeys::Report.key(),          report);
-    medData->setMetaData(medMetaDataKeys::Protocol.key(),        protocol);
-    medData->setMetaData(medMetaDataKeys::Comments.key(),        comments);
-    medData->setMetaData(medMetaDataKeys::Status.key(),          status);
-
-    medData->addMetaData(medMetaDataKeys::FilePaths.key(),      filePaths);
-
+    if (medData)
+    {
+        // PATIENT
+        //PatientId
```

Comment:
I sorted metadata update as in the medMetaDataKeys file, and displayed the not-used parameters in order to see if we want to add new updates